Source/WebCore/platform/graphics/FontCache.cpp

```diff
@@ -128,19 +129,32 @@ struct FontCache::FontDataCaches {
 #endif
 };
 
+static FontCache*& fontCacheForCurrentThread()
+{
+    static ThreadSpecific<FontCache*>* fontCache;
+    static std::once_flag onceFlag;
+    std::call_once(onceFlag, [] {
+        fontCache = new ThreadSpecific<FontCache*>;
```

Comment:
Can we do ThreadSpecific<FontCache> instead, to remove the need to allocate/delete FontCache?

Reply:
We want to be able to explicitly destroy the FontCache on demand (to guarantee shutdown ordering), so I don't think this can work.